roles/dbid-logger/tasks/main.yml

```diff
@@ -17,7 +17,9 @@
   ansible.builtin.shell: |
     set -o pipefail
     sqlplus -s / as sysdba <<EOF
-    SELECT dbid FROM v\$database;
+    SET HEADING OFF
+    SET FEEDBACK OFF
+    SELECT TRIM(dbid) FROM v\$database;
```

Comment:
a trim() of an implicit cast of a number has no effect.  The leading space is caused by sql*plus's formatting.